### DISCUSSION TITLE
photoprism --site-url not working

### DISCUSSION THREAD

**Discussion by Angelus187** [photoprism/photoprism#1405] [Q&A] [answered]
Good day all,

I'm new to this. I installed the Photoprism docker on UNRAID but did not change the default URL. Now, I've tried the command _photoprism --site-url "http://my.url.com:2342/"_ and _photoprism --site-url="http://my.url.com:2342/"_ to no avail.

Can anyone help me with this? Please?

Angelus

**Comment by lastzero** [comment, marked as answer]
You need to provide an action argument like "start" after it. Otherwise, nothing happens and you only see the help.

**Comment by Angelus187** [comment]
Thank Michael.

I'll try and let you know.

Le ven. 2 juill. 2021 19 h 54, Michael Mayer ***@***.***> a
écrit :
